Minutes/2023/2023_08_07-minutes.md

```diff
@@ -0,0 +1,211 @@
+Meeting Minutes of 07 August 2023
+===
+
+:date: **Date:** 07 August 2023
+
+### :bust_in_silhouette: Participants
+
+- Ege Korkan
+- Cristiano Aguzzi
+- Fady Salama
+- Justus Fries
+- Idil Sezgin
+
+
+:writing_hand: **Scribe:** Ege Korkan
+
+### Tutorial Series
+
+Ege Korkan: we can talk about the tutorial series and then on the TPAC
```

Comment:
```suggestion
Ege Korkan: we can talk about the tutorial series at TPAC
```

Reply:
actually, the first one is the correct intention. However, there is a small typo that I will provide separately.

```suggestion
Ege Korkan: we can talk about the tutorial series and then about the TPAC
```